README.md

```diff
@@ -7,9 +7,9 @@ through [Spicy](https://github.com/zeek/spicy).
 
 Currently, the following analyzers are included:
 
-- DHCP
-- DNS
-- HTTP
+- DHCP [1]
+- DNS [1]
+- HTTP [1]
```

Comment:
Since GH markdown doesn't support footnotes and the references here aren't clickable, maybe we should just add the footnote text in parentheses, e.g.,

```suggestion
- DHCP (replaces corresponding Zeek analyzer)
- DNS (replaces corresponding Zeek analyzer)
- HTTP (replaces corresponding Zeek analyzer)
```

Reply:
Had it like that first, but am preferring the pseudo-footnote for brevity; have another one coming too